### PR TITLE
docs: DOC-161: Beta tag and note about syncing with taxonomy 

### PR DESCRIPTION
**Pull request by caitlinwheeless**
Updated to:

- Add beta tag and note to external taxonomy section
- Add a note about configuring external storage when using taxonomy
- Add a bit of clarity about two types of taxonomy

### CODE REVIEW

**Review by jombooth — APPROVED** [review]
this is great :sparkling_heart: :ship:

File: docs/source/templates/taxonomy.md [hunk 1]
```diff
@@ -96,9 +96,19 @@ The `apiUrl` must be accessible to Label Studio. You can accomplish this in seve
...
+2. [Follow these instructions](storage) to set up cloud storage for Label Studio. This should be a separate connection from your other storage connections.
+
+    <div class="admonition note"><p class="admonition-title">note</p><p>Do not sync this storage connection. If you do, you will need to delete the task that is automatically created when syncing the taxonomy.</p></div>
+
+3. When configuring your labeling interface, format your `apiUrl` using the appropriate URL format for your cloud service provider: `gs://`, `s3://`, `azure-blob://`. 
 
-    [Follow these instructions to set up cloud storage for Label Studio](storage). You can then format your `apiUrl` using the appropriate URL format for your cloud service provider: `gs://`, `s3://`, `azure-blob://`. However, you should set up a storage connection for your taxonomy source that is separate from data you will be syncing. (See the note below.)
 
 !!! warning Note on setting up cloud storage
     You can connect Label Studio to an external cloud environment where you store your taxonomy, and then call the taxonomy using the appropriate cloud-specific URL (`gs://`, `s3://`, `azure-blob://`).
```
Comment (hlomzik) [inline docs/source/templates/taxonomy.md]: This looks like an excess paragraph now as it fully duplicates the main instruction as a note.

File: docs/source/templates/taxonomy.md [hunk 2]
```diff
@@ -96,9 +96,19 @@ The `apiUrl` must be accessible to Label Studio. You can accomplish this in seve
...
+* The URL points to your private cloud environment. This is the most secure option. See [Using cloud storage with an external taxonomy](#Using-cloud-storage-with-an-external-taxonomy) below.  
+
+### Using cloud storage with an external taxonomy 
+  
+1. Save your taxonomy in a separate bucket from your task data. 
+
+    If you are unable to use a separate bucket for your taxonomy, see the workaround below. 
+2. [Follow these instructions](storage) to set up cloud storage for Label Studio. This should be a separate connection from your other storage connections.
+
+    <div class="admonition note"><p class="admonition-title">note</p><p>Do not sync this storage connection. If you do, you will need to delete the task that is automatically created when syncing the taxonomy.</p></div>
```
Comment (hlomzik) [inline docs/source/templates/taxonomy.md:108]: Can we use `!!!` format here to not use too specific html and class names?

**Review by hlomzik — APPROVED** [review]

File: docs/source/templates/taxonomy.md (hunk 2 above)
Comment (caitlinwheeless) [reply docs/source/templates/taxonomy.md:108]: Unfortunately, no. I tried, but it wasn't rendering with it was a child of an `<li>`. Will need to ask Hugues if this is something we can fix.

File: docs/source/templates/taxonomy.md (hunk 1 above)
Comment (caitlinwheeless) [reply docs/source/templates/taxonomy.md]: Agreed - removed.

**Pull request merged**